packages/widget-v2/src/utils/misc.ts

```diff
@@ -1,6 +0,0 @@
-import crypto from 'crypto';
```

Comment:
can we check if we use `crypto` package anywhere, if not let's remove it from our deps

Reply:
@codingki we don't using crypto anywhere else, but also it's not an explicit dep anywhere in our package.json it's a core node_module which is being polyfilled in our vite.config.ts